src/programs/console_test/console_test.cpp

```diff
@@ -450,9 +579,9 @@ void MyTestApp::TestStarToBinaryFileConversion()
   fread(star_file_from_binary_file, 1, star_from_binary_size_in_bytes, current_file);
   fclose(current_file);
 
-  for (long byte_counter = 63; byte_counter < original_star_size_in_bytes; byte_counter++) // start at byte 63, the before that is a comment which includes the time written
+  for (long byte_counter = header_bytes_to_ignore; byte_counter < original_star_size_in_bytes; byte_counter++) // start at byte header_bytes_to_ignore, the before that is a comment which includes the time written
   {
-    if (original_star_file[byte_counter] != star_file_from_binary_file[byte_counter]) FailTest;
+    if (original_star_file[byte_counter] != star_file_from_binary_file[byte_counter]) { std::cerr << "failed on byte" << byte_counter << std::endl; FailTest;}
```

Comment:
I think this should be okay, as long as there in the StarFileWriter to point out the \n is a signal in a test. I'll make the change, as there is a "silent" return at the top of the function that should (for now) return an int. icpc didn't care, but gcc did.

The only difference I think, is rather than the continue statement, I'll do something like
```c++
bool ignoring_first_line = true;
if (ignoring_first_line)
  // We've reached the end of the first line which breaks ... blah blah
  if (original_star_file[byte_counter] == '\n') ignoring_first_line = false;
else
  do error checking

```